### PR TITLE
Remove unused /dashboard/config/* directories

### PR DESCRIPTION
**Pull request by TurnerRiley**
As part of [this PR](https://github.com/code-dot-org/code-dot-org/pull/53886), we were deprecating unused code realted to state school data. In reviewing this, Dave spotted some more directories that could be removed so this PR removes them and uses of them:
- `dashboard/config/other_curriculum_offerings`
- `dashboard/config/ap_cs_offerings`
- `dashboard/config/ap_school_codes`

## Links
Jira ticket: [here](https://codedotorg.atlassian.net/jira/software/c/projects/ACQ/boards/64?modal=detail&selectedIssue=ACQ-1002&assignee=60d62161f65054006980bd71)
PR this builds on: [here](https://github.com/code-dot-org/code-dot-org/pull/53886)

## Testing story
Drone build passing.

## Follow-up work
The removal of `ap_cs_offerings` and `ap_school_codes` will require migrations to remove content from our databases as well.

## PR Checklist:
- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: aws/redshift/views/census_details.sql [hunk 1]
```diff
@@ -9,8 +9,6 @@ with csa as
       when cso.school_code is null and ss.school_type in ('public', 'charter') then 0 -- only include known status for public and charter schools, as we don't have data for private schools
     end as teaches_csa
   from analysis.school_stats ss
-    left join dashboard_production.ap_school_codes sc on sc.school_id = ss.school_id
```
Comment (sureshc) [inline aws/redshift/views/census_details.sql]: Has some from @code-dot-org/red-team reviewed the changes to these Redshift-related queries?

Comment (bakerfranke) [reply aws/redshift/views/census_details.sql]: Thanks for calling this out @sureshc.  I need some time to review.

**Review by bakerfranke — COMMENTED** [review]
@TurnerRiley Can you verify that the sql file you've modified here (and others in the same directory) are not actually executed on a regular basis, but just stored in this file path for "safe keeping"?

While it's true we don't use these files or SQL anymore to drive the access report if there's a chance we're "unplugging" something it may have some ramifications on analytics.  Additionally, if we're deprecating these things on dashboard, I'd like to save them for posterity and move them over to our analytics repo since we may need to reconstruct some things.

**Comment by cat5inthecradle** [comment]
@TurnerRiley Would it make sense to split this into two PR's? That would also make it easier to extract the code we might want to archive in a more discoverable location.

**Comment by TurnerRiley** [comment]
> @TurnerRiley Would it make sense to split this into two PR's? That would also make it easier to extract the code we might want to archive in a more discoverable location.

Good call yes I don't know a lot about the details of the uses of the `census_details.sql` file. Just to clarify what you're asking, have this PR be the removal of the `.csv` files and not change the SQL code (saving that for another PR where we can extract the code we may want to save)?

**Review by jordan-springer — APPROVED** [review]
This all looks fine to me, my only concern would be around the updates to views that were proposed earlier and since removed.

**Review by davidsbailey — APPROVED** [review]

**Pull request merged**